ansible/roles/cluster/tasks/initialize.yml

```diff
@@ -0,0 +1,69 @@
+---
+- name: check if init is needed
```

Comment:
Curious about a master that is disconnected from the pool and needs to be rejoined? Or does this just require it get re-flashed?

Reply:
Good point, havent thought about that. Right now, this will only ever run on the first master, so if one of the other masters requires to be rejoined (e.g. after a crash), then it will work fine. However, if the first master is a fresh install within an existing cluster environment, this will indeed cause an issue.

I will keep this in mind and see if I can come up with a solution.